hack/release-yaml.sh

```diff
@@ -1,13 +1,33 @@
-#!/bin/bash
+#!/bin/bash -eu
+set -o pipefail
+# Generate manifests for deployment on Kubernetes
 
-TAG=${1?$(git describe --abbrev=0)}
-RELEASES="pkg/linode-bs/deploy/releases/"
+# A tag name _must_ be supplied as the first argument
+TAG="${1}"
+if [[ -z "${TAG}" ]]; then
+    echo "Tag name to release must be supplied as the first argument"
```

Comment:
To allow this error message to print, you will have to remove `-u` above

Reply:
Right! I can make that change.